AdyenSession/AdyenSession.swift

```diff
@@ -57,7 +57,7 @@ public final class AdyenSession {
         public let identifier: String
         
         /// Country Code
-        public let countryCode: String
+        public let countryCode: String?
```

Comment:
reminder to check if this one is not needed since it's a public property

Reply:
@erenbesel @descorp, when are you planning to release it?